AndroidTest/app/build.gradle

```diff
@@ -55,10 +40,13 @@ android {
     sourceSets {
         main {
             java.srcDirs = ['src/main/java',
-                            "$scriptLocation/../../sync-core/src/test/java/",
-                            "$scriptLocation/../../sync-core/src/main/java/",
-                            "$scriptLocation/../../sync-android/src/test/java/",
-                            "$scriptLocation/../../sync-android/src/main/java/"]
+                            "$scriptLocation/../../sync-core/src/main/java",
```

Comment:
sync-core main is only built as part of the app, its not included in the test source sets